packages/react-components/react-tabs/src/components/Tab/useTab.ts

```diff
@@ -35,6 +36,7 @@ export const useTab_unstable = (props: TabProps, ref: React.Ref<HTMLElement>): T
 
   const innerRef = React.useRef<HTMLElement>(null);
   const onTabClick = useEventCallback(mergeCallbacks(onClick, (event: SelectTabEvent) => onSelect(event, { value })));
+  const onTabFocus = useEventCallback(mergeCallbacks(onFocus, (event: SelectTabEvent) => onSelect(event, { value })));
```

Comment:
Should you maybe move the select callback into its own const above so that any changes to it don't need to be applied twice later on?